### DISCUSSION TITLE
Ajout d'une politique

### DISCUSSION THREAD

**Discussion by BafeDede23** [pep-un/Oxomium#16] [Q&A] [answered]
Comment faire pour ajouter une politique interne ou encore une norme internationale autre que celles déjà présentes dans le logiciel?
Aussi, ISO27001 comporte 114 mesures de sécurité, mais il me semble ne voir que 26 mesures dans le logiciel. S'agit-il de 26 parmi les 114 ou alors 26 autres fonction de la norme ? Si c'est parmi les 114, où sont les autres ? Si c'est fonction de la norme, sur quoi se base ces mesures ?

Merci!

**Comment by pep-un** [comment]
Bonjour,

L'ajout d'une politique interne est parfaitement possible. Il faut pour cela passer par le les fixtures Django pour importer votre politique sous forme de fichier yaml (exemple: https://github.com/pep-un/Oxomium/tree/main/conformity/fixture). Il est aussi possible de créer la politique à la main, item par item, dans le back office de django.

Pour l'ISO27k1, je n'ai pas décrit chaque item de la norme mais uniquement les titres et sous-titres. Il faudrait profiter de la sortie de la version 2022 pour être plus précis et reprendre chaque item.

**Reply by pep-un** [reply, marked as answer]
La mise en forme ne correspond effectivement pas aux points de contrôle formulé par l'iso27001:2013. 

Je vais voir pour mètre à jour la 2013 et créé la version 2022.

Toutes aide est la bienvenue :)

**Comment by BafeDede23** [comment]
Toute aide que je puisse fournir, ce sera avec joie!

**Reply by pep-un** [reply]
Si tu a un csv/xls des exigences iso27001, je peux l'intégrer sans problème !
